lib/cell.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/ship'
```

Comment:
The `Cell` class doesn't reference `Ship` anywhere, so you shouldn't need this line.